File: C/section06/S06E08.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    int number;

    printf("Please enter an integer number: \n");
    scanf("%d", &number);

    if (number % 2 == 0) {
        if (number > 0) {
        printf("\nThe number is even and positive.\n");
        } else {
            printf("\nThe number is even and negative.\n");
        }
    } else {
        if (number > 0) {
        printf("\nThe number is odd and positive.\n");
        } else {
            printf("\nThe number is odd and negative.\n");
        }
    }

    return 0;
}

File: Python/section06/S06e08.py
number = int(input ("Please enter a number : "))

if number % 2 == 0:
    if number > 0:
        print("\nThe number {0} is even and positive.".format(number))
    else: 
        print("\nThe number {0} is even and negative.".format(number))
else:
    if number > 0:
        print("\nThe number {0} is odd and positive.".format(number))
    else:
        print("\nThe number {0} is odd and negative.".format(number))